specifyweb/frontend/js_src/lib/components/FormCommands/ShowTransactions.tsx

```diff
@@ -114,10 +116,12 @@ export function ShowLoansCommand({
   return typeof data === 'object' ? (
     <Dialog
       buttons={commonText.close()}
-      header={commonText.transactions()}
+      header={interactionsText.interactions()}
+      icon={<span className="text-blue-500">{icons.chat}</span>}
```

Comment:
(Just an observation) 

There are a plethora of places in Specify which use `<span className="text-blue-500">{icons.someIcon}</span>` specifically for Dialog icons. 
It may be worth creating a utility function (perhaps in https://github.com/specify/specify7/blob/production/specifyweb/frontend/js_src/lib/components/Atoms/Icons.tsx) which would accept a JSX element (or a `keyof icons`) and returns a span with that className and the corresponding icon. 

We could then replace all instances where this dialog icon convention is used with a call to the utility function.

Reply:
good idea. @grantfitzsimmons can you tackle that?

though, while we have the dialog icon be blue in many places, it's not always the case - probably just accidental inconsistency

thus, would it make more sense to modify `<Dialog>` to accept an icon name in the `icon` prop, and it would make that icon blue automatically?

Reply:
That may be more sensible, although _always_ making the dialog icon blue may not be the desired behavior in all cases. 
For instance, and most notably, when using a `dialogIcon`:

https://github.com/specify/specify7/blob/956a14bb75cb986a53f6f04988eca19fd66deae5/specifyweb/frontend/js_src/lib/components/Atoms/Icons.tsx#L134-L147

Reply:
@maxpatiiuk @melton-jason see my latest commits and let me know what you think